### PR TITLE
Fix orientation and pose for enu

### LINKED ISSUE #121: Bug | Orientation is in the wrong coordinate frame
Hello,

We are using a VN-100T, and we are getting a `-π` roll when the IMU is flat on a surface. In other words, Z is pointing down.
image
From looking a little at the code we found a couple of issues:
1) https://github.com/dawonn/vectornav/blob/933af893ab91986110dab2528a6fa6c913f4f932/vectornav/src/vn_sensor_msgs.cc#L181 Orientation is always converted to ENU even if `use_enu` is false. 
2) The above conversion actually breaks the orientation and causes the above mentioned issue. From our tests, the quaternion that arrives is already in ENU, so this conversion is redundant (nevertheless a conversion should exist if `use_enu` is false to turn it into NED).
3) Pose is always converted to ECEF, and afaik should also be in ENU.
https://github.com/dawonn/vectornav/blob/933af893ab91986110dab2528a6fa6c913f4f932/vectornav/src/vn_sensor_msgs.cc#L309.

I don't mind issuing a PR, just would like to get assurance I am not missing something.

### PR DESCRIPTION
**Pull request by nakai-omer**
fixed #121

### CODE REVIEW

**Comment by dawonn** [comment]
How did you test this, and did you ensure that both NED and ENU work as expected?

**Comment by nakai-omer** [comment]
@dawonn We have tested ENU both in simulation and on our real robot. NED we haven't really tested to be honest. Will test and update.

File: vectornav/src/vn_sensor_msgs.cc [hunk 1]
```diff
@@ -114,6 +114,22 @@ class VnSensorMsgs : public rclcpp::Node
   }
 
 private:
+  void convert_to_enu(const vectornav_msgs::msg::CommonGroup::SharedPtr msg_in, sensor_msgs::msg::Imu msg_out) const
```
Comment (james-ward) [inline vectornav/src/vn_sensor_msgs.cc]: This does not take the output message by reference, and so none of the fields are actually set.
```suggestion
  void convert_to_enu(const vectornav_msgs::msg::CommonGroup::SharedPtr msg_in, sensor_msgs::msg::Imu &msg_out) const
```

Comment (james-ward) [reply vectornav/src/vn_sensor_msgs.cc]: Otherwise I can confirm this works after testing on a real robot.

**Pull request merged**